src/sql/statement.rs

```diff
@@ -836,13 +820,7 @@ fn handle_create_view(
     })
 }
 
-/// Remove dependencies on external state: inline schemas in files,
-/// fetch schemas from registries, and so on, so that we can generate a
-/// fully-specified `create-sql` to persist in the catalog.
-///
-/// The `Statement` returned from this function should be able to be planned
-/// without any asynchronicity or non-determinism.
-async fn purify_statement(mut stmt: Statement) -> Result<Statement, failure::Error> {
+pub async fn purify_statement(mut stmt: Statement) -> Result<Statement, failure::Error> {
```

Comment:
Can this be `pub(crate)` instead of `pub` ?

Anyway, if it's being made public, it should have a comment IMO (the comment can just be "see the one in lib.rs" or vice-versa)

Reply:
It’s a public function in a private module, so is not actually public.